docs/release-notes/keyboard_from_live.rst

```diff
@@ -0,0 +1,22 @@
+:Type: Live Environment
+:Summary: Use keyboard layout configuration from the Live system
+
+:Description:
+    Until now Anaconda, it was required to specify keyboard layout for the Live environment.
+    With this change, live system itself is responsible for the keyboard configuration and
+    Anaconda just reading the configuration from the live system for the installed system.
+
+    This works only if it is not specified by user and only Gnome Shell environment is
+    supported so far. Spins should be added later.
```

Comment:
```suggestion
    The live keyboard layout is used automatically only if the user does not specify it manually.
    At this moment, only Gnome Shell environment is supported.
```

I'd leave out any hopes and plans for the future.

Reply:
Honestly, we should address these at least for KDE because the current situation is really bad.

Reply:
I agree we should improve that. I just feel that promising it this way might not be the best idea. But that's my docs instincts...